test/test_testing.py

```diff
@@ -1398,6 +1398,21 @@ def test_matching_per_channel(self):
             fn()
 
 
+class TestMakeTensor(TestCase):
+    @dtypes(torch.bool, torch.uint8, torch.int8, torch.int16, torch.int32, torch.int64)
+    def test_low_high_bool_integral(self, dtype, device):
+        shape = (100,)
+        eps = 1e-4
+
+        actual = torch.testing.make_tensor(shape, dtype=dtype, device=device, low=-(1 - eps), high=1 - eps)
+        expected = torch.zeros(shape, dtype=dtype, device=device)
+
+        torch.testing.assert_close(actual, expected)
+
+
+instantiate_device_type_tests(TestMakeTensor, globals(), only_for="cpu")
```

Comment:
It'd be OK to also run this on other device types, but makes sense to limit it to cpu for speed

Reply:
I needed to use `instantiate_device_type_tests` to enable the `dtypes` decorator. Can make this more exhaustive.